examples/util/base.js

```diff
@@ -777,17 +777,32 @@ emotionAnalysisModels: [{
 }],
 
 speechCommandModels: [{
-  modelName: 'KWS CNN (TFLite)',
-  format: 'TFLite',
-  modelId: 'kws_cnn_tflite',
-  modelSize: '3.4MB',
-  inputSize: [1, 16000],
-  outputSize: [1, 1, 12],
-  sampleRate: 16000,
-  modelFile: '../speech_commands/model/kws_cnn.tflite',
-  labelsFile: '../speech_commands/model/labels.txt',
-  intro: 'Use Convolutional Neural Networks (CNNs) for a small-footprint keyword spotting (KWS) task.',
-  paperUrl: 'https://www.isca-speech.org/archive/interspeech_2015/papers/i15_1478.pdf'
+    modelName: 'KWS CNN (TFLite)',
+    format: 'TFLite',
+    modelId: 'kws_cnn_tflite',
+    modelSize: '3.4MB',
+    inputSize: [1, 16000],
+    outputSize: [1, 1, 12],
+    sampleRate: 16000,
+    modelFile: '../speech_commands/model/kws_cnn.tflite',
+    labelsFile: '../speech_commands/model/labels.txt',
+    intro: 'Use Convolutional Neural Networks (CNNs) for a small-footprint keyword spotting (KWS) task.',
+    paperUrl: 'https://www.isca-speech.org/archive/interspeech_2015/papers/i15_1478.pdf'
+  }, {
+    modelName: 'KWS DNN (TFLite)',
+    format: 'TFLite',
+    modelId: 'kws_dnn_tflite',
+    modelSize: '320kB',
+    inputSize: [1, 250],
+    outputSize: [1, 12],
+    sampleRate: 16000,
+    modelFile: '../speech_commands/model/kws_dnn.tflite',
```

Comment:
@miaobin Please update [README.md](https://github.com/intel/webml-polyfill/blob/master/examples/speech_commands/model/README.md), add the link of this kws_dnn model and its descriptions, thanks

Reply:
could we use OpenVINO models instead?

Reply:
> could we use OpenVINO models instead?

Yes, the model have been converted to both tflite and OpenVINO IR format.

Reply:
> @miaobin Please update [README.md](https://github.com/intel/webml-polyfill/blob/master/examples/speech_commands/model/README.md), add the link of this kws_dnn model and its descriptions, thanks

OK, I will update the README.

Reply:
> Yes, the model have been converted to both tflite and OpenVINO IR format.

Great. Then could this example use the OpenVINO IR?

Reply:
> > Yes, the model have been converted to both tflite and OpenVINO IR format.
> 
> Great. Then could this example use the OpenVINO IR?

OK, you think we'll use the both or just the OpenVINO IR?

Reply:
> Great. Then could this example use the OpenVINO IR?

I would suggest OpenVINO IR first.

Reply:
> > Great. Then could this example use the OpenVINO IR?
> 
> I would suggest OpenVINO IR first.

OK, I'll update this commit.